minidec/main.rs

```diff
@@ -59,6 +67,7 @@ fn main() {
         //println!("Local Variable info: {:?}", rfn.locals);
         let mut writer: IRWriter = Default::default();
         writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ff);
+        writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ffm);
```

Comment:
I think I see the problem. I need to change that std::io::Write to std::fmt::Write. Can you try that out too?